src/views/Home.jsx

```diff
@@ -1,16 +1,23 @@
 import './Home.css';
-
-export function Home({ data, setListPath }) {
+import { SingleList } from '../components';
+export function Home({ data, setListPath, path }) {
```

Comment:
See other comment - the path comes from the list itself.

```suggestion
export function Home({ data, setListPath }) {
```